week-1-ts-ecosystem/w01e01-typescript-basics/package.json

```diff
@@ -1,5 +1,13 @@
 {
   "name": "w01e01-typescript-basics",
   "version": "1.0.0",
-  "scripts": {}
+  "scripts": {
+    "dev": "nodemon --exec ts-node src/main.ts",
```

Comment:
z tego go kojarzę to wystarczy samo:
`nodemon src/main.ts`

`nodemon` z automatu się orientuje że jest `ts-node` zainstalowany i że uruchamia wtedy go dla plików `.ts`